pkg/process/checks/host_info.go

```diff
@@ -15,9 +15,9 @@ import (
 	"strings"
 	"time"
 
-	model "github.com/DataDog/agent-payload/v5/process"
 	"google.golang.org/grpc"
 
+	model "github.com/DataDog/agent-payload/v5/process"
```

Comment:
I think this import should stay where it was, since it's an import from an external repo